### PR TITLE
Add support for webpack 5

### PR DESCRIPTION
**Pull request by johnnyreilly**
based on the advice given by @liangchunn here: https://github.com/Realytics/fork-ts-checker-webpack-plugin/pull/166

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,15 +27,19 @@
     "url": "https://github.com/johnnyreilly/fork-ts-checker-notifier-webpack-plugin/issues"
   },
   "dependencies": {
+    "fork-ts-checker-webpack-plugin": "^1.0.0-alpha.0",
```
Comment (liangchunn) [inline package.json]: I feel like this should be moved into `peerDependencies` instead. Reasoning is that a user might want a different version of fTCWP, and the user can override fTCNWP's tTCWP version.

**Review by liangchunn — COMMENTED** [review]
Look great! We also might want to remove node6 from the test matrix.

File: package.json (hunk 1 above)
Comment (johnnyreilly) [reply package.json]: Cool - will do

Comment (johnnyreilly) [reply package.json]: It's now listed as a `devDependency` of:

```
    "fork-ts-checker-webpack-plugin": "^1.0.0-alpha.1",
```

and a `peerDependency` of:

```
    "fork-ts-checker-webpack-plugin": "*"
```

Sound about right?

Comment (liangchunn) [reply package.json]: We might want to use something like `1.x` or `^1.0.0` since this upgrade won’t support fTCWP below 1.0

Comment (johnnyreilly) [reply package.json]: Good shout - just pushed.  Any other thoughts before I publish an alpha?

Comment (liangchunn) [reply package.json]: Nope, LGTM!

**Comment by johnnyreilly** [comment]
published

**Pull request merged**